python/pyspark/ml/tests/test_image.py

```diff
@@ -24,7 +24,7 @@
 
 class ImageFileFormatTest(SparkSessionTestCase):
     def test_read_images(self):
-        data_path = "data/mllib/images/origin/kittens"
+        data_path = "python/test_support/images/kittens"
```

Comment:
does pyspark have the test resource directory? If not I think it's fine to reuse the data files in `/data/mllib/images/...`

Reply:
yeah, pyspark doesn't have test resource directory.

Reply:
Revert this change.